### PR TITLE
Merge discvr-21.3 to develop

### PR DESCRIPTION
**Pull request by bbimber**

### CODE REVIEW

**Comment by bbimber** [comment]
@labkey-jeckels If I merge into develop, will these make it into 21.5?

File: singlecell/src/org/labkey/singlecell/pipeline/singlecell/AbstractRDiscvrStep.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.labkey.api.sequenceanalysis.pipeline.PipelineContext;
+import org.labkey.api.sequenceanalysis.pipeline.PipelineStepProvider;
+import org.labkey.api.singlecell.pipeline.AbstractSingleCellPipelineStep;
+import org.labkey.api.util.PageFlowUtil;
+
+import java.util.Collection;
+
+abstract public class AbstractRDiscvrStep extends AbstractSingleCellPipelineStep
+{
+    public static String CONTINAER_NAME = "ghcr.io/bimberlabinternal/rdiscvr:latest";
```
Comment (labkey-jeckels) [inline singlecell/src/org/labkey/singlecell/pipeline/singlecell/AbstractRDiscvrStep.java]: Typo

**Review by labkey-jeckels — APPROVED** [review]
We've already branched for 21.5. We haven't tagged 21.5.0 yet, but we won't be bulk-merging develop changes into the release branch

File: singlecell/src/org/labkey/singlecell/pipeline/singlecell/AbstractRDiscvrStep.java (hunk 1 above)
Comment (bbimber) [reply singlecell/src/org/labkey/singlecell/pipeline/singlecell/AbstractRDiscvrStep.java]: thanks

**Comment by bbimber** [comment]
@labkey-jeckels OK. I was thinking 21.5 was a major release, but it's actually 21.3, 21.7 and 21.11, right?

**Comment by labkey-jeckels** [comment]
@bbimber correct, 21.7 is the next big one. We'll branch on or around July 1, and ship 21.7.0 around July 15.

**Pull request merged**